docs/Biconomy AA Stack/Account/installation.md

```diff
@@ -57,6 +57,10 @@ The common package is needed by our accounts package as another dependency.
 
 import { BiconomySmartAccount, BiconomySmartAccountConfig } from "@biconomy/account"
 
+:::info
+Note that we are using Etheres to pass the signer to the initialization here. However you can get the signer from many different SDKs available such as Web3.js, Wagmi React hooks, And auth providers like Particle, Web3Auth, Magic, and more.
```

Comment:
Theres written, should be ethers. general typo


Reframe : We are utilizing Ethers to pass the signer during initialization. However, it's worth mentioning that you have the flexibility to obtain the signer from various other SDKs. Some popular options include Web3.js, Wagmi React hooks, as well as authentication providers like Particle, Web3Auth, Magic, and many others.